cmd/state/main.go

```diff
@@ -43,6 +45,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	if runtime.GOOS == "windows" {
+		osv, err := sysinfo.OSVersion()
+		if err != nil {
+			logging.Debug("Could not retrieve os version info: %v", err)
+		}
+		if osv.Major < 10 {
```

Comment:
This will be a nil pointer panic if it errored above.

Reply:
Thanks for catching.